### ISSUE TITLE
Dataset request: <dataset title>

### ISSUE THREAD

**Issue by Kimsterz** [datagovsg/datagovsg-datasets#837]
### Please provide the name of the dataset or describe the dataset that you're requesting:
Aircraft Noise complaint dataset w/postal codes

### Do you know which agency provides/can provide this data?
*( Refer to this link for a full list of agencies: https://data.gov.sg/organization )*
- Agency for science, technology and research
- Civil aviation authority of Singapore 
- Ministry of Defence 
- National Environment Agency
### Please provide the fields expected in this dataset:

month | postal code | nature of complaint|time of complaint lodged|
------|---------|-------------
2017-09|730371|aircraft flying before 6am| 6:05

### How frequently would you need this dataset to be updated?
Example: 
- [X] *Test*

- [ ] Annually
- [X ] Monthly
- [ ] Weekly
- [ ] Daily
- [ ] Hourly
- [ ] Real-time

### Can you give a short explanation on how you're planning to use this dataset?

So basically, I'd like to know which area in Singapore is heavily affected by aircraft movements (both civil and military) and map the complaint distribution out to see if it correlates to the flight path undertaken by the aircraft. 
I'm using the requested dataset as part of my research for my capstone project (under the Singapore Institute of Technology).

**Comment by lennardl** [comment]
@Kimsterz We do not have that data, you might need to approach the government agencies directly for it
